recipe_scrapers/breadtopia.py

```diff
@@ -0,0 +1,36 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class Breadtopia(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "breadtopia.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
```

Comment:
Could please add test coverage for this `category` call?

It's currently returning `Recipes` which isn't necessarily useful information but since it's included in the sites outputs and covered by the scraper we should have test backup.

Reply:
@jknndy fixed, thanks